app/src/main/dist/config/crate.yml

```diff
@@ -294,9 +294,10 @@
 # allocated on a single node.
 #index.routing.allocation.total_shards_per_node: -1
 
-# When using local gateway a particular shard is recovered only if there can
+# When shards are recovered and AllocationIds have not been assigned, use
```

Comment:
The existing doc string mentions Elasticsearch internals as well. Plus, this config setting is Elasticsearch internal anyways. And finally, it was broken because it had been renamed in Elasticsearch.

I think we should get rid of it in the long run.